<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="originalPathLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Image Path</string>
    </property>
   </widget>
   <widget class="QLabel" name="destinationPathLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination Image Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="originalPathLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>461</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="destinationPathLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>461</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="originalPathPushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="destinationPathPushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="subtitleLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Image Dataset Transformations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>179</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="grayScaleCheckBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grayscale Images</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rotateCheckBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Images 0ยบ - 90ยบ - 180ยบ - 270ยบ</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="flipCheckBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip Images</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cropCheckBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop Images</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="windowSizeLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>999999</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="csvCheckBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CSV Files with Coordinates for Crop</string>
    </property>
   </widget>
   <widget class="QLabel" name="windowSizeLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="csvLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>461</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="csvLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CSV Files Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csvPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>470</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="thinPlateSplinesCheckBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Thin Plate Splines Transformations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generatePushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>560</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="resizeCheckBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize Images</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="widthLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>296</y>
      <width>67</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="heightLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
